db/migrate/20191222215207_add_sort_name_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddSortNameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :sort_name, :string
+    add_index :users, :sort_name
+
+    # Re-save all users so that sort_name field is set for each.
+    User.find_each { |user| user.save }
```

Comment:
While it may not be something that we do often, it's not actually wrong.  Even the Rails guide says:

> The main purpose of Rails' migration feature is to issue commands that modify the schema using a consistent process. Migrations can also be used to add or modify data. This is useful in an existing database that can't be destroyed and recreated, such as a production database.

That said, I should have made this an up/down type of migration to ensure that it's reversible.